15-7-1_and_15-c-1/15-7-1_and_15-c-1_checks.Rmd

```diff
@@ -0,0 +1,117 @@
+---
+title: "15-7-1 and 15-c-1 checks"
+author: "Katie Uzzell"
+date: "22/12/2022"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+```{r load-packages, include=FALSE}
+library(tidyr)
+library(dplyr)
+library(ggplot2)
+library(kableExtra)
+```
+
+```{r include= FALSE}
+source('15-7-1_15-c-1_example_config.R')
+
+old_data <- read.csv("https://sdgdata.gov.uk/sdg-data/en/data/15-7-1.csv")
+
+date <- Sys.Date()
+new_data <- read.csv(paste0("Example_Output/", date, "_update_15-7-1_15-c-1", ".csv")) 
+
+latest_year <- max(new_data$Year)
+
+```
+
+## 15-7-1 and 15-c-1 update summary
+
+Data for `r min(new_data$Year)` to `r latest_year` have been created and 
+saved as `r paste0(date, "_15-7-1_and_15-c-1", ".csv")` in the 15-1-1 Output folder.  
+    
+### Check configurations
+Please check that the information below is as you expect. In particular, pay 
+attention to the time since data were saved, to check that previously 
+downloaded data have not accidentally been used.
+    
+These settings can all be adjusted in `config.R`
+    
+Files used to create the new data were downloaded and saved in 
+15-7-1_and_15-c-1/`r input_folder`.  
+  
+```{r include= FALSE}
+filename_save_time <- file.info(paste0(getwd(),"/Example_Input/", filename), full.names = TRUE)$ctime[1]
+time_since_data_saved <- round(difftime(Sys.time(), filename_save_time, units = "days"), 2)
+
+```
+<br>
+
+**Seizure data from Border Force**  
+
+Seizure data are in the file `r filename`.   
+
+This file was saved at `r filename_save_time`: It has been 
+`r time_since_data_saved` days since the file was saved in 
+`r input_folder`  
+
+The tab used is called
+`r tabname`. 
+    
+<br>
+
+
+### Check data
+The charts below show the new data (just created) plotted on top of the old data,
+which has been downloaded from the live site. If there are unexpected 
+discrepancies in the lines please investigate the reasons for these discrepancies.
+    
+```{r, echo=FALSE, fig.width=7, fig.height=15}
+new_data <- new_data %>% 
+mutate(Year = as.integer(Year),
+         dataset = "new") 
+
+old_data$Import.type = str_to_sentence(old_data$Import.type)
+
+old_data$Import.type[old_data$Import.type == ""] <- "Total number of seizures"
+
+joined_data <- old_data %>% 
+  mutate(dataset = "old") %>% 
+  bind_rows(new_data)
+
+plot <- joined_data %>% 
+  ggplot(.,
+         aes(x = Year,
+             y = Value,
+             colour = dataset)) +
+  scale_x_continuous (breaks=min (joined_data$Year):max (joined_data$Year)) +
+  geom_point() +
+  geom_line() +
+  facet_wrap(vars(Import.type),
+             nrow = 6,
+             ncol = 2) +
+  theme_bw()  
+
+```
+
+
+```{r, echo = FALSE, fig.height = 8, fig.width = 10}
+suppressMessages(print(plot))
+```
+
```

Comment:
You can add a small section here that lists the NA values from that variable we created in the main script. Something like:

### Missing/ suppressed values

If any values in the original data were suppressed, these are given in the table below. Consider adding a footnote or other explanation for partially suppressed values in the indicator's metadata. For example, when some quarters were suppressed, but an annual value is still calculated, it should be interpreted with caution.

```{r, echo = FALSE}
kable(select_NAs, caption = "Missing values")
```
This will print a table of the rows with NAs before they were removed. Not sure what it does if there were no removals though. Worst case will just show an empty line

Reply:
Done